packages/testcontainers/src/containers/RegTestContainer/Masternode.ts

```diff
@@ -184,6 +184,32 @@ export class MasterNodeRegTestContainer extends RegTestContainer {
     }, timeout, 100, 'waitForAnchorRewardConfrims')
   }
 
+  async waitForPriceValid (fixedIntervalPriceId: string, timeout = 30000): Promise<void> {
+    // @ts-expect-error
```

Comment:
What's the error expected here?

Reply:
I have removed this line in my latest code